pkg/registry/controller/etcd/etcd.go

```diff
@@ -117,8 +117,8 @@ func (r *StatusREST) New() runtime.Object {
 }
```

Comment:
A unrelated question, does anyone know why the folder is called "controller" instead of "replicationcontroller"?